File: studio/v1/step_context/fetch-default/fetch-default.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/console
$sid    = "ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$step_context = $twilio->studio->v1->flows("FWaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")
                                   ->engagements("FNaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")
                                   ->steps("FTaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")
                                   ->stepContext()
                                   ->fetch();

print($step_context.accountSid);

File: studio/v1/step_context/fetch-default/fetch-default.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Studio.V1.Flow.Engagement.Step;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var stepContext = StepContextResource.Fetch(
            pathFlowSid: "FWaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
            pathEngagementSid: "FNaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
            pathStepSid: "FTaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
        );

        Console.WriteLine(stepContext.AccountSid);
    }
}
